Makefile

```diff
@@ -71,20 +70,22 @@ clean:
 
 .PHONY: test
 test: auto-format
-	@tox -e py35,flake,isort
+	# NB: Remove once pyyaml 4.2 officially released
+	PIP_PRE=1 tox -e py35,flake,isort
 	@shellcheck snap/wrappers/*
 
 git_rev:
 	@echo $(GIT_REV)
 
 dev: clean
-	tox -e conjure-dev
+	# NB: Remove once pyyaml 4.2 officially released
+	PIP_PRE=1 tox -e conjure-dev
 	@echo "Run 'source conjure-dev/bin/activate' to enter the dev venv"
 
 # Fix some of the python formatting preferred by pylint
 auto-format:
-	@tox -e isort -- isort -rc -m 3 conjureup test tools
-	@tox -e isort -- autopep8 --in-place --recursive conjureup test tools
+	PIP_PRE=1 tox -e isort -- isort -rc -m 3 conjureup test tools
```

Comment:
going to remove this env var in next commit and just use PIP_PRE=1 make test for now